tests/test_capabilities.py

```diff
@@ -119,29 +112,25 @@ def conditional_branch_on_bit():
 
 
 class ConfigurableQisVisitor(BasicQisVisitor):
-    def __init__(self, matrix: List[bool], profile: str = "AdaptiveExecution"):
-        BasicQisVisitor.__init__(self, profile)
-        self._matrix = matrix
+    def __init__(self, profile: str = "AdaptiveExecution", **kwargs):
+        BasicQisVisitor.__init__(self, profile, **kwargs)
 
     def visit_qiskit_module(self, module):
         BasicQisVisitor.visit_qiskit_module(self, module)
-        self._module.use_static_qubit_alloc(self._matrix[0])
-        self._module.use_static_result_alloc(self._matrix[1])
```

Comment:
nit: It doesn't seem like this does anything now, can you use BasicQisVisitor directly?